packages/api/src/external/ehr/athenahealth/command/process-patients-from-appointments-sub.ts

```diff
@@ -181,7 +173,9 @@ async function getAppointments({
   fromDate,
   toDate,
 }: GetAppointmentsParams): Promise<{ appointments?: Appointment[]; error?: unknown }> {
-  const { log } = out(`AthenaHealth getAppointments - cxId ${cxId} practiceId ${practiceId}`);
+  const { log } = out(
+    `AthenaHealth getAppointments - cxId ${cxId} practiceId ${practiceId} departmentIds ${departmentIds}`
```

Comment:
do we need to have the list of departments on each log? could log it once and not have it on the prefix